src/docker/eventListener.ts

```diff
@@ -54,13 +58,14 @@ export class EventListener {
     this.poller = new IntervalPoller(
       "pollDockerEvents",
       () => {
-        this.listenForEvents();
+        void this.listenForEvents();
       },
```

Comment:
I was on the fence about this one while working on https://github.com/confluentinc/vscode/pull/2787 - should we leave this without the `void` and (in a follow-up branch) update `IntervalPoller` to always [`await` its callback](https://github.com/confluentinc/vscode/blob/b50768ddc4992274e740b44caab3f25e8ee0ab0b/src/utils/timing.ts#L172-L175)?